static/src/javascripts/projects/membership/contributions-recurring-tab.js

```diff
@@ -457,8 +462,18 @@ const changeContributionAmountSubmit = (): void => {
             },
         }
     )
-        // eslint-disable-next-line no-use-before-define
-        .then(recurringContributionTab(true))
+        .then(resp => {
+            hideLoader();
+            if (resp.status === 200) {
```

Comment:
The response will definitely be 200 on success will it? No chance of a 304 or anything?

Reply:
That's a good question. I'd expect the API to return 200 unless an actual error happens. That could change in future I suppose. I'd hope it wouldn't return a not modified response though.